Tensile/TensileCreateLibrary.py

```diff
@@ -1131,6 +1131,22 @@ def param(key, value):
       
       param("best-solution", True)
 
+
+def generateManifest(manifestFile: Path, contents: List[str]) -> None:
+    """Generates tensile manifest.
```

Comment:
This is a simple stylistic question, but should we capitalize Tensile throughout our doc-strings and web-documentation? Either way, just want to remain consistent.

Reply:
Sure lets do that